en/airframes/adding_a_new_frame.md

```diff
@@ -162,8 +162,21 @@ S: 0 3      0  20000 -10000 -10000  10000
 
 ```
 
-### Getting the new airframe to show in QGroundControl
+
+
+## VTOL-specific Configuration
```

Comment:
Does this section make sense here? Is there a better place?

Reply:
Yes, looks good. Unless there is a VTOL-specific tuning page?

Reply:
Good point thanks. I'll add a link to the tuning docs.